src/LoginFormContainer.test.jsx

```diff
@@ -0,0 +1,88 @@
+
+import React from 'react';
+
+import { fireEvent, render } from '@testing-library/react';
+
+import LoginFormContainer from './LoginFormContainer';
+import { useDispatch, useSelector } from 'react-redux';
+
+jest.mock('react-redux');
+
+describe('LoginFormContainer', () => {
+  const dispatch = jest.fn();
+  beforeEach(() => {
+    dispatch.mockClear();
+
+    useDispatch.mockImplementation(() => dispatch);
+
+    useSelector.mockImplementation((selector) => selector({
+      loginFields: {
+        email:'test@test.com',
+        password:'1234',
+      },
+      accessToken: given.accessToken,
+    }));
+  });
+
+  context('when logged out', () => {
+    given('accessToken', () => '');
+    it('renders input controls', () => {
+      const { getByLabelText } = render(
+          <LoginFormContainer />
+      );
```

Comment:
```suggestion
      const { getByLabelText } = render(<LoginFormContainer />);
```

한 줄로 해도 될 것 같아요.
만약에 줄바꿈을 해주고 싶다면 괄호를 2번 감싸주세요. 그래야 컴포넌트 다음에 콤마가 안붙어요.